zmon-controller-app/src/main/java/org/zalando/zmon/config/ControllerProperties.java

```diff
@@ -25,6 +25,8 @@
     public int emailTokenLength = 15;
     public int emailPort = 465;
 
+    public String grafanaHost = "";
```

Comment:
Are you sure you don't need getters/setters for this field? I see that it's public, I'm asking because other fields in this class have corresponding methods

Reply:
added